velox/expression/tests/ExpressionFuzzer.cpp

```diff
@@ -536,6 +538,39 @@ std::vector<core::TypedExprPtr> ExpressionFuzzer::generateRegexpReplaceArgs(
   return inputExpressions;
 }
 
+std::vector<core::TypedExprPtr> ExpressionFuzzer::generateSwitchArgs(
```

Comment:
The current code treat Swtich as a regular function in the pool to be randomly chosen from. I wonder whether we would like to make a `generateSwitchExpression()` to be called in `generateExpression()` so that it can be tested more frequently? (cc @mbasmanova)

Reply:
As a follow up to this, I considered implementing something like this for both switch and conjuncts but any solution would end up adding more specialized code to the implementation and add complexity that could have been avoided. Instead I decided to implement a more generic framework for specifying function names that we want to prefer. See #3388 for more details on the proposed design.